ethsigner/signer/hashicorp/src/main/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommand.java

```diff
@@ -39,6 +43,35 @@
   private static final Integer DEFAULT_PORT = Integer.valueOf(DEFAULT_PORT_STRING);
   private static final Long DEFAULT_TIMEOUT = Duration.ofSeconds(10).toMillis();
 
+  static class HashicorpTrustStore implements PkcsStoreConfig {
+
+    @Option(
+        names = "--tls-server-trust-store-file",
```

Comment:
not sure I love the naming - but then, not sure I love any of our naming in this area.